eve/main.yml

```diff
@@ -229,7 +229,7 @@ stages:
         env:
           <<: *multiple-backend-vars
           <<: *global-env
-          CI_CEPH: enabled
+          CI_CEPH: true
```

Comment:
I think this will need to be quoted because it registers as a bool instead of a string. I checked on the eve build and ends up like `CI_CEPH=True` in the build environment.